### PR TITLE
ChatOps: update format command / Update CI container

### PR DESCRIPTION
**Pull request by rinx**
### Description:

I added yaml-formatter to /format command of ChatOps.
I've updated CI container.

### Related Issue:

Nothing.

### How Has This Been Tested?:

Nothing.

### Environment:

- Golang Version: 1.14
- Docker Version: 19.03.5
- Kubernetes Version: 1.16.3
- NGT Version: 1.9.1

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [X] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [X] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [X] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensure all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: dockers/ci/base/Dockerfile [hunk 1]
```diff
@@ -26,8 +27,39 @@ COPY versions versions
...
+
+RUN make helm/install
+RUN make kind/install
+RUN make valdcli/install
+
+RUN curl -L https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl -o /usr/local/bin/kubectl \
+    && chmod a+x /usr/local/bin/kubectl
+
+RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh \
+    | sh -s -- -b $(go env GOPATH)/bin v1.23.6
```
Comment (kpango) [inline dockers/ci/base/Dockerfile]: I think it's better to define version as ENV or ARG

**Comment by rinx** [comment]
Okay.
I've just added *dockfmt*.

**Comment by kpango** [comment]
LGTM. thank you for your  great job!!
/format
/approve

**Pull request merged**